docs/diagrams/notes/NotesCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+
+participant "AddNoteCommandParser" as Parser
```

Comment:
participant should have colons in front, i.e. `:AddNoteCommandParser`, ditto for the remaining participant

moreover, your sequence diagram has a footer box which can otherwise be removed by adding `hide footbox`

perhaps, also add colors to the boxes just like how @lordidiot did for his sequence diagram